main/inc/lib/message.lib.php

```diff
@@ -782,6 +782,9 @@ public static function send_message_simple(
                         get_lang('CopyOfMessageSentToXUser'),
                         $userInfo['complete_name']
                     ).' <br />'.$message;
+                    if(!isset( $drhInfo['user_id']) and isset( $drhInfo['id'])){
+                         $drhInfo['user_id'] = $drhInfo['id'];
+                    }
```

Comment:
That code is not needed. UserManager::getDrhListFromUser returns only "id"

Better to fix:

```
self::send_message_simple(
    $drhInfo['id'],
```